x/yarpcmeta/service.go

```diff
@@ -56,9 +58,88 @@ func (m *service) procs(ctx context.Context, body interface{}) (*procsResponse,
 
 func (m *service) introspect(ctx context.Context, body interface{}) (*introspection.DispatcherStatus, error) {
 	status := m.disp.Introspect()
+	status.Procedures.BasicIDLOnly()
 	return &status, nil
 }
 
+type idlsQuery struct {
+	EntryPoint string   `json:"entryPoint,omitempty"`
+	Selection  []string `json:"selection,omitempty"`
+}
+
+type idlFile struct {
+	FilePath   string   `json:"filePath"`
+	SHA1       string   `json:"sha1"`
```

Comment:
Same comments as other PR re: SHA1, RawContent. Also note this maps to `Module` in the other PR, naming?